userguide/sound_classifier/README.md

```diff
@@ -0,0 +1,55 @@
+# Sound Classifier
+
+Given a sound, the goal of the Sound Classifier is to assign it to
+one of a pre-determined number of labels, such as baby crying, siren,
+or dog barking. This Sound Classifier is not intended to be used for
+speech recoginition.
+
+#### Loading Data
+
+For example purposees, we will use the ESC-10 dataset. This dataset
+contain ten classes. Each class has 40 examples with five seconds of
+audio per example.
+
+**Please note:** the ESC-10 dataset is part of a larger [ESC-50 dataset](https://github.com/karoldvl/ESC-50)
+dataset. In order to use the ESC-10, you will need to download the
+[ESC-50 dataset](https://github.com/karoldvl/ESC-50). These two datasets have different licenses.
+Depending on your use case you may be able to the larger ESC-50
+dataset, see their [license section](https://github.com/karoldvl/ESC-50#license) for details.
+
+#### Introductory Example
+```python
+import turicreate as tc
+from os.path import basename
+
+# Load the audio data and meta data.
+data = tc.load_audio('./ESC-50/audio/')
```

Comment:
Don't we have a with_path = True on this like with load images? It reduces the need for the base name for the join.

Reply:
We do have a `with_path=True` parameter on `tc.load_images(...)`. I've also added it `tc.load_audio(...)` where it is also on by default; I'm using it here.

We still need the `basename` apply before the join because in the meta data CSV only the filenames are stored. `load_images` and `load_audio` both put the full path in the SFrame, when the `with_path` parameter is true.